src/settings-ui/Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -17,133 +17,133 @@
+                                                Grid.RowSpan="2"
+                                                x:Uid="Enable_ColorFormat"
+                                                AutomationProperties.HelpText="{x:Bind Name}"
+                                                HorizontalAlignment="Right"
+                                                Margin="0,0,56,0"/>
 
-                                    <Button Background="Transparent"
-                                            FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                            HorizontalAlignment="Right"
-                                            VerticalAlignment="Center"
-                                            Grid.RowSpan="2"
-                                            x:Uid="More_Options_Button"
-                                            Content="&#xEDE3;">
-                                        <Button.Flyout>
-                                            <MenuFlyout>
-                                                <MenuFlyoutItem IsEnabled="{x:Bind CanMoveUp}" Icon="Up" x:Uid="MoveUp" Click="ReorderButtonUp_Click"/>
-                                                <MenuFlyoutItem IsEnabled="{x:Bind CanMoveDown}" x:Uid="MoveDown" Click="ReorderButtonDown_Click">
-                                                    <MenuFlyoutItem.Icon>
-                                                        <FontIcon Glyph="&#xF0AE;" />
-                                                    </MenuFlyoutItem.Icon>
-                                                </MenuFlyoutItem>
-                                            </MenuFlyout>
-                                        </Button.Flyout>
-                                        <ToolTipService.ToolTip>
-                                            <TextBlock x:Uid="More_Options_ButtonTooltip" />
-                                        </ToolTipService.ToolTip>
-                                    </Button>
-                                </Grid>
-                            </DataTemplate>
-                        </ListView.ItemTemplate>
-                    </ListView>
-                </controls:SettingsGroup>
-            </StackPanel>
+                                <Button Background="Transparent"
+                                        FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                        HorizontalAlignment="Right"
+                                        VerticalAlignment="Center"
+                                        Grid.RowSpan="2"
+                                        x:Uid="More_Options_Button"
+                                        Content="&#xEDE3;">
+                                    <Button.Flyout>
+                                        <MenuFlyout>
+                                            <MenuFlyoutItem IsEnabled="{x:Bind CanMoveUp}" Icon="e74a" x:Uid="MoveUp" Click="ReorderButtonUp_Click"/>
```

Comment:
This crashes the app.. Remove the Icon property and add:

  <MenuFlyoutItem.Icon>
                                                    <FontIcon Glyph="&#xe74a;" />
                                                </MenuFlyoutItem.Icon>

Reply:
Yeah, it's annoying how `Icon` doesn't seem to accept the hex code and `Icon="Down"` doesn't work either 😒
Fixed both.